reports/JSV.md

```diff
@@ -2,8 +2,6 @@
 
 # All validators fail this test
```

Comment:
Would be nice if this text is not there when no tests are failed by all validators.